r2/r2/lib/eventcollector.py

```diff
@@ -123,6 +123,12 @@ def vote_event(self, vote, old_vote=None,
             for name, value in private_data.iteritems():
                 event.add(name, value)
 
+        if context_data:
+            referrer_url = context_data.get('referer')
+            if referrer_url:
+                event.add("referrer_url", referrer_url)
```

Comment:
This is a good point from @xiongchiamiov.

In quarantine events we get referrer_domain and the referrer_url as separate fields (although I'm not sure if these get populated). Seems like there are two options:

1. Send referrer_domain and referrer_url for just vote events
2. Send referrer_domain and referrer_url for all events